recipes/loguru/all/conanfile.py

```diff
@@ -0,0 +1,147 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.build import check_min_cppstd
+
+required_conan_version = ">=1.54.0"
```

Comment:
```suggestion
required_conan_version = ">=1.53.0"
```
It does not need 1.54.0.